app/src/main/java/cm/aptoide/pt/ads/MoPubAdsService.java

```diff
@@ -0,0 +1,41 @@
+package cm.aptoide.pt.ads;
+
+import cm.aptoide.pt.crashreports.CrashReport;
+import cm.aptoide.pt.install.InstalledRepository;
+import rx.Scheduler;
+import rx.subscriptions.CompositeSubscription;
+
+public class MoPubAdsService {
+
+  private static final String WALLET_PACKAGE = "com.appcoins.wallet";
+
+  private final CompositeSubscription subscriptions;
```

Comment:
we only have 1 subscription here and we are not unsubscribing it since It should probably live through the application scope.